disk-mapper/internal/recoveryserver/server_test.go

```diff
@@ -71,59 +77,13 @@ func TestServe(t *testing.T) {
 
 func TestRecover(t *testing.T) {
 	testCases := map[string]struct {
-		initialMsg message
-		keyMsg     message
-		wantErr    bool
+		masterKey string
+		salt      string
+		wantErr   bool
```

Comment:
Would probably make sense to refactor this function now to work on KMS URIS, so we don't have to refactor again once eKMS gets impemented

Reply:
Refactored and introduced a KMS stub.